nrf_wifi/fw_if/umac_if/inc/fmac_structs_common.h

```diff
@@ -114,6 +115,10 @@ struct nrf_wifi_fmac_reg_info {
 	unsigned char alpha2[NRF_WIFI_COUNTRY_CODE_LEN];
 	 /** Forcefully set regulatory. */
 	bool force;
+	/** Regulatory channels count*/
+	unsigned int reg_chan_count;
+	/** Regulatory channel attributes */
+	struct nrf_wifi_get_reg_chn_info reg_chan_info[MAX_NUM_REG_CHANELS] __NRF_WIFI_ALIGN_4;
```

Comment:
What about the size of `nrf_wifi_get_reg_chn_info`, it should also be multiple of 4, is that handled?

Reply:
struct nrf_wifi_get_reg_chn_info reg_chan_info[MAX_NUM_REG_CHANELS] __NRF_WIFI_ALIGN_4";" is required other wise the image itself crash on boot because of alignment issue. No other issue was observed realated to alignment.

Reply:
Sure, it just might be working because it's already aligned to 4, but both the start_address and length should be 4 bytes aligned, so, need to explicitly make sure they are aligned.

Reply:
The issue is being observed no more even without alignment for "reg_chan_info". So removed __NRF_WIFI_ALIGN_4 for reg_chan_info element.